rlImGui.cpp

```diff
@@ -99,7 +99,13 @@ static void rlImGuiNewFrame(float deltaTime)
     }
     else 
     {
-        io.DisplayFramebufferScale = ImVec2(1.0f, 1.0f);
+
+#if !__APPLE__
+      io.DisplayFramebufferScale = ImVec2(1.0f, 1.0f);
+#else
+      Vector2 windowScaleDpi = GetWindowScaleDPI();
+      io.DisplayFramebufferScale = ImVec2(windowScaleDpi.x, windowScaleDpi.y);
```

Comment:
This should not be apple specific right? I think we should be able to call this on all platforms.

Reply:
Yes you're right. The function is from raylib, so it should work on all platforms. I've used it on both MacOS and Windows in my project https://github.com/PhantomCloak/cengine-2d and works nicely with different monitor & DPI setups.